README.md

```diff
@@ -2,8 +2,8 @@
 
 # Project Name
 
-> My Portfolio
--It's project built using HTML and CSS to see how github flow works.
+> LEADEROARD APPLICATION
+-It's project built using HTML, JavaScript and CSS to see how github flow works.
 
 ## Built With
```

Comment:
- [ ] Nice Job building this Amazing project, however, you also need to add instructions to run the project locally, please.

### EXAMPLE
```
### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/bigmosi/Leader-board/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/bigmosi/Leader-board.git
```

```
- Once the files are on your machine, open the _Leader-board_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set-Up

## 🛠 Installation & Set Up

1. Install Node Modules

   ```sh
   npm install
   ```

```
2. Start the development server

   ```sh
   npm start
   ```

3. Open [http://localhost:8080](http://localhost:8080) to view it in the browser.
```
```
```